connect/src/routes/cctp/automatic.ts

```diff
@@ -179,9 +179,21 @@ export class AutomaticCCTPRoute<N extends Network>
     let nativeGasAmount = 0n;
 
     if (nativeGasPerc > 0.0) {
+      const dcb = await request.toChain.getAutomaticCircleBridge();
+      let maxSwapAmount = await dcb.maxSwapAmount();
```

Comment:
Can `getAutomaticCircleBridge()` return undefined? Even if there is a slight possibility, it'd be good to check 👍

Reply:
The function to get it will throw if it's undefined and I'm following the pattern used across the SDK here for getting the chain's protocol implementation